test/testautomation_intrinsics.c

```diff
@@ -4,7 +4,7 @@
 
 #ifndef NO_BUILD_CONFIG
 /* Disable intrinsics that are unsupported by the current compiler */
-#include <build_config/SDL_build_config.h>
+#include <SDL_build_config.h>
```

Comment:
Should this use `#include "SDL_build_config.h"` like the other tests?

Reply:
Let's do that for consistency.